src/ParseServer.js

```diff
@@ -210,7 +211,16 @@ class ParseServer {
 
     Config.validate(AppCache.get(appId));
     this.config = AppCache.get(appId);
-    hooksController.load();
+    if (hooksRefreshIntervalMs > 0) {
+      (function repeat() {
+        setTimeout(() => {
```

Comment:
so get rid of most of the triggers.js code and have each hook be a key with a given ttl and a do a separate query to fetch each hook?